main/webapp/modules/core/langs/translation-en.json

```diff
@@ -1,6 +1,10 @@
 {
     "name": "English",
     "core-index/slogan": "A power tool for working with messy data",
+    "core-index/definition" : "OpenRefine is a power tool for working with messy data. Use it to improve data consistency,link it to data registries like Wikidata, augment it with data from other sources, transform it into different formats for other tools to consume, and contribute it to back to the original sources. OpenRefine is not a web service but a desktop app that runs on your own computer,so you can process sensitive data with privacy.",
```

Comment:
Perhaps the prefix shouldn't be `core-index` but rather `core-about` since this is not on the index page but the about page?